components/automate-chef-io/content/docs/iam-v2-guide.md

```diff
@@ -215,6 +215,11 @@ To do this, open any policy from the _Policies_ list, then select **Members**.
 Select **Add Members** to open the list of candidate local users and teams.
 Near the bottom of the page, select the **Add Member Expression** button.
 
+{{% info %}}
+The member expression dialog guides you through creating a member expression to alleviate the tedium of getting the syntax just right.
+But the next few paragraphs explain the syntax, should you prefer to add members via the API.
+{{% /info %}}
```

Comment:
PR Bonus: The above block should have been added when the member expression dialog changes went in recently.